tests/rkt_net_test.go

```diff
@@ -338,78 +346,89 @@ func TestNetDefaultRestrictedConnectivity(t *testing.T) {
 	f("--net=default-restricted")
 }
 
+type CommonPortFwdCases struct {
+	HttpGetIP     string
+	RktArg        string
+	ShouldSucceed bool
+}
```

Comment:
since you introduced a dedicated struct we also could:
```
type commonPortFwdCasesTest []CommonPortFwdCases

func (ct commonPortFwdCasesTest) Execute(t *testing.T) {
  // all the code below without superfluous indentation
}

func NewCommonPortFwdCasesTest(cases []CommonPortFwdCases) Test {
  return CommonPortFwdCasesTest(cases)
}
```